atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ColumnRangeSelections.java

```diff
@@ -20,9 +20,13 @@
 import com.google.common.base.Preconditions;
 import com.palantir.common.persist.Persistable;
 
-public class ColumnRangeSelections implements Serializable {
+public final class ColumnRangeSelections implements Serializable {
+    private ColumnRangeSelections() {
+        // Utility class
+    }
+
     public static ColumnRangeSelection createPrefixRange(byte[] prefix, int batchSize) {
-        byte[] startCol = Preconditions.checkNotNull(prefix).clone();
+        byte[] startCol = Preconditions.checkNotNull(prefix, "prefix cannot be null").clone();
```

Comment:
Talked to @gsheasby about this. We'd prefer to keep the Preconditions in with the stupid messages than remove them.